### PR TITLE
fixes borgs sending the vox shipjack

### PR DESCRIPTION
**Pull request by Intigracy**
fags

### CODE REVIEW

**Comment by Allspice** [comment]
FUG

File: code/game/shuttles/vox.dm [hunk 1]
```diff
@@ -62,6 +62,10 @@ var/global/datum/shuttle/vox/vox_shuttle = new(starting_area=/area/shuttle/vox/s
 	link_to(vox_shuttle)
 	.=..()
 
+/obj/machinery/computer/shuttle_control/vox/attack_ai(mob/user)
+	to_chat(user, "<span class='warning'>This computer has a firewall preventing you from interacting with it.</span>")
```
Comment (9600bauds) [inline code/game/shuttles/vox.dm]: Missed opportunity to add in a funny message like "You can't make sense of this terminal's programming, it's all chicken scratch."

Comment (Intigracy) [inline code/game/shuttles/vox.dm]: Thanks for being funnier than I am, will fix.

Comment (ihadtoregisterforthis) [inline code/game/shuttles/vox.dm]: But borgs can understand pidgin, can't they?

Comment (9600bauds) [inline code/game/shuttles/vox.dm]: The joke is that vox programming is shit

Comment (Intigracy) [inline code/game/shuttles/vox.dm]: they understand the written/spoken language, but they don't understand the code obviously

File: html/changelogs/Intivoxjack.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+author: Intigracy
+delete-after: True
+changes: 
+- bugfix: "Silicons can no longer interact with the vox shipjack shuttle computer."
```
Comment (Exxion) [inline html/changelogs/Intivoxjack.yml]: It's skipjack, not shipjack
Important difference

**Comment by ghost** [comment]
Isn't there already a variable to say that borgs can't interact with shuttle computers that have it?

var/allow_silicons, set it to 0, and silicons can't use the shuttle

**Comment by clusterfack** [comment]
He's right, use allow silicons

**Comment by 9600bauds** [comment]
fix it

**Comment by Intigracy** [comment]
k

**Pull request merged**